sentence_transformers/sparse_encoder/search_engines.py

```diff
@@ -168,8 +172,8 @@ def semantic_search_elasticsearch(
     Performs semantic search using sparse embeddings with Elasticsearch.
 
     Args:
-        query_embeddings: PyTorch COO sparse tensor containing query embeddings
-        corpus_embeddings: PyTorch COO sparse tensor containing corpus embeddings
+        query_embeddings_decoded: List of query embeddings in format [[("token": value), ...], ...]
+        corpus_embeddings_decoded: List of corpus embeddings in format [[("token": value), ...], ...]
```

Comment:
Let's briefly try and explain how users can get this, e.g. `embeddings = model.encode(texts)` followed by `model.decode(embeddings)`. Or maybe an Example in the docstring?